alphastats/gui/pages/04_Analysis.py

```diff
@@ -35,6 +32,8 @@
     """
 st.markdown(styl, unsafe_allow_html=True)
 
+# TODO put everything in the session state for a given parameter set?
+# or is caching functionality the way to go here?
```

Comment:
Caching is the way to go. But only chache calculations, not plots. The expensive part about plots is the rendering in the browser and we can't cache that away.